api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -231,6 +235,9 @@ public void setUp() throws IOException {
             .country(COUNTRY)
             .zipCode(ZIP_CODE));
 
+    // TODO: this needs to be set in createAccountAndDbUserWithAffiliation() instead of here.  Why?
```

Comment:
not a change, just an observation of the current state

Reply:
not currently.  My assumption is that these test classes will change quite a lot, so I plan to revisit then or soon after